parsec/mca/device/cuda/device_cuda_module.c

```diff
@@ -575,23 +575,15 @@ parsec_cuda_module_init( int dev_id, parsec_device_module_t** module )
     }
 
     if( show_caps ) {
-        parsec_inform("GPU Device %-8s: %s [capability %d.%d] %s\n"
```

Comment:
not sure I understand the need to change the device info.

Reply:
The main goal was to add the amount of actually allocated memory (and the total capacity), the rest of the reformatting is just cosmetic to reduce the number of output lines, the old format was pretty annoying on Frontier with 8GPUs per node.